apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.spec.tsx

```diff
@@ -38,6 +38,6 @@ describe('Installed Service Account Card', () => {
     });
 
     await screen.findByText('Google Service Account Details');
-    await screen.findByText('Active');
+    await screen.findByText('Access correctly configured');
```

Comment:
this feels a bit weird to me... WDYT about sth like `Configuration verified` or `Access verified`?